### PR TITLE
Removed old bundle deployer interaction code which is no longer needed

### PR DESCRIPTION
**Pull request by hatched**
With the new recordSet work we no longer send the YAML to the deployer and wait for the updates. We now send all the deploy commands individually from the GUI.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Can you export an environment?
Can you import an environment?
Good!

File: app/assets/javascripts/bundle-import-helpers.js [hunk 1]
```diff
@@ -1,195 +0,0 @@
...
-    },
-
-    /**
-    Export the YAML for the environment.
-
-    @method exportYAML
-    @param {Database} db The app Database with access to the
-      NotificationList.
-    */
-    exportYAML: function(db) {
```
Comment (hatched) [inline app/assets/javascripts/bundle-import-helpers.js]: This functionality was still needed so it was moved to the deployer bar where it was primarily used from.

**Comment by makyo** [comment]
Code LGTM, QAing now

**Comment by makyo** [comment]
Weird QA thing.  Deploying a bundle to an environment (import) works well, but when I try to click export, even after refreshing, I get:

```
models.js:2314 Uncaught TypeError: Cannot set property 'to' of undefined
(anonymous function)	@	models.js:2314
Y.Base.create._addMachinesToServices	@	models.js:2312
Y.Base.create.exportDeployer	@	models.js:2202
Y.Base.create._exportFile	@	deployer-bar.js:873
Y.Base.create.exportFile	@	deployer-bar.js:864
delegate.notifySub	@	event-delegate.js:173
```

Haven't traced yet.

**Comment by hatched** [comment]
This bug is fixed in PR #729 - It's because we are excluding the GUI service when generating the service list but didn't exclude it when generating the machines.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2508/
Test PASSed.

File: app/assets/javascripts/bundle-import-helpers.js (hunk 1 above)
Comment (mitechie) [inline app/assets/javascripts/bundle-import-helpers.js]: Okay sounds like the other branch needs  to land before this one can be
thoroughly Qa'd

**Comment by makyo** [comment]
:+1: QA okay - looks like #729 fixed most of my complaints - I'll keep an eye out for weirdness surrounding machines being blank

**Comment by hatched** [comment]
Thanks for the reviews on this! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**